### PR TITLE
Update changelog for v1.9.0

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

**Review by Falke-Design — APPROVED** [review]

**Comment by jonkoops** [comment]
Just noticed that all the previous releases include extended links for user handles and PR. Going to revise that, please hold the merge.

**Comment by jonkoops** [comment]
@Falke-Design I added some proper markup for the links and such, could you give this another look?

**Review by Falke-Design — APPROVED** [review]
Looks good :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,82 @@ Leaflet Changelog
...
+* Fix "listener not found" warning when setting `maxBounds` ([#8168](https://github.com/Leaflet/Leaflet/pull/8168))
+* Remove "listener not found" warning ([#8234](https://github.com/Leaflet/Leaflet/pull/8234) by [@Falke-Design](https://github.com/Falke-Design))
+* Extend `Events.listens` to search for specific function ([#8161](https://github.com/Leaflet/Leaflet/pull/8161) by [@Falke-Design](https://github.com/Falke-Design))
+* Add `noMoveStart` option to `panTo` ([#6685](https://github.com/Leaflet/Leaflet/pull/6685) by [@Chivano](https://github.com/Chivano))
+* Add `FeatureCollection` handling to `geometryToLayer` ([#8163](https://github.com/Leaflet/Leaflet/pull/8163) by [@Falke-Design](https://github.com/Falke-Design))
+
+### 🙌 Accessibility
+* Improve `Tooltip` accessibility (focus and voice over) ([#8247](https://github.com/Leaflet/Leaflet/pull/8247) by [@alekzvik](https://github.com/alekzvik))
+* Fix links in accessibility guide ([#8198](https://github.com/Leaflet/Leaflet/pull/8198) by [@Malvoz](https://github.com/Malvoz))
+* Remove `role="presentation"` from image tiles ([#8172](https://github.com/Leaflet/Leaflet/pull/8172) by [@Malvoz](https://github.com/Malvoz))
```
Comment (Malvoz) [inline CHANGELOG.md:28]:
```suggestion
* Remove redundant `role="presentation"` from image tiles ([#8172](https://github.com/Leaflet/Leaflet/pull/8172) by [@Malvoz](https://github.com/Malvoz))
```

**Review by Malvoz — APPROVED** [review]

**Review by mourner — APPROVED** [review]

**Pull request merged**